src/main/webapp/result.html

```diff
@@ -0,0 +1,72 @@
+ <!-- Copyright 2020 Google LLC
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+        https://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<!DOCTYPE html>
+<html>
+
+<head>
+    <!--info for browser -->
+    <meta charset="UTF-8">
+    <meta name="keywords" content="error, logs, search, stacktrace">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- favicon -->
+    <link rel="shortcut icon" type="image/x-icon" href="images/gSuite.png" />
+    <!-- title -->
+    <title>Error analyzer</title>
+
+    <!-- add css -->
+    <link rel="stylesheet" href="result.css">
+    <!-- add fontawesome icons -->
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css" 
+    integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU" crossorigin="anonymous">
+</head>
+
+<body onload="changePage()">
+    <header id="body-header" class="backgroundColor">
+        <!-- search bar -->
+        <div>
+            <span>
+                <input type="text" id="searchBar" class="shadow inputDimension font" name="searchBar">
+                <button onclick="search()" ><i class="fas fa-search"></i></button>
+            </span>
+        </div>
+        <!-- choose file  -->
+        <div>
+            <span>
+                <input type="text"  id="fileName" class="shadow inputDimension font" name="fileName">
+                <!-- if user is changing file then show results starting from 1st page -->
+                <button onclick="changePage()" >submit</button>
+            </span>
+            <!-- log or error container -->
+            <span>
+                <input type="radio" id="logs" name="display" value="logs" checked>logs
+                <input type="radio" id="errors" name="display" value="errors">errors
+            </span>
+        </div>
+    </header>
+    <!-- header -->
+
+    <!-- show result -->
+    <div>
+        <div id="listingTable"></div>
+    </div>
+    <!-- navigate between pages -->
+    <footer class="backgroundColor text-center">
+        <div id="navigation">
+            <a href="javascript:prevPage()" id="btnPrev">Prev</a>
```

Comment:
Didn't know this works :p
I used to add hyperlink style in css to buttons for this.